core/timeline_inc.php

```diff
@@ -79,14 +79,14 @@
 
 				echo '<div class="btn-group">';
 				$t_url_params['days'] = $f_days + 7;
-				$t_href = $t_url_page . '?' . http_build_query( $t_url_params );
+				$t_href = htmlspecialchars( $t_url_page, ENT_QUOTES, 'UTF-8' ) . '?' . http_build_query( $t_url_params );
```

Comment:
Escaping should be done via *string_attribute()* API function.

Can you explain the rationale for using ENT_QUOTES vs the default of ENT_COMPAT ?

Also, is there a reason to force UTF-8 charset ? (for the record, this is the default since PHP 5.4)